pennylane/beta/queuing/measure.py

```diff
@@ -227,3 +227,44 @@ def circuit():
     meas_op = MeasurementProcess(Probability, wires=qml.wires.Wires(wires))
     QueuingContext.append(meas_op)
     return meas_op
+
+
+def state(wires):
+    r"""Quantum state in the computational basis.
+
+    This function accepts no observables and instead instructs the QNode to return its state. The
+    ``wires`` argument must specify all wires used in the device, as shown in the example below.
+    This guarantees that the returned state is pure.
+
+    **Example:**
+
+    .. code-block:: python3
+
+        dev = qml.device("default.qubit", wires=2)
+
+        @qml.qnode(dev)
```

Comment:
This currently works with the new QNode, but I guess we should put the examples how we expect them to work when we merge into core?

```python
from pennylane.beta.tapes import qnode
import pennylane as qml

dev = qml.device("default.qubit", wires=2)

@qnode(dev)
def circuit():
    qml.Hadamard(wires=1)
    return qml.state(wires=range(2))

circuit()
```

Reply:
Ok, I've updated to use beta :grin: